catalog/plugin/catalog-plugin-metacard-validation/pom.xml

```diff
@@ -84,7 +84,7 @@
                                         <limit>
                                             <counter>INSTRUCTION</counter>
                                             <value>COVEREDRATIO</value>
-                                            <minimum>0.9</minimum>
+                                            <minimum>0.8</minimum>
```

Comment:
I honestly don't think we should be setting anything up above the default unless we have a good reason to. If we satisfy the defaults, we should just remove the entire section.